assets/src/edit-story/components/library/panes/pageTemplates/pageTemplatesPane.js

```diff
@@ -127,7 +133,7 @@ function PageTemplatesPane(props) {
       <PaneInner>
         {customPageTemplates && (
           <>
-            {savedTemplates && (
+            {savedTemplates && !isDefaultPage(currentPage) && (
```

Comment:
I feel it would be better to make the button disabled (coordinated with UX) rather than hide and show - the whole panel jumps around when you add something to an empty page while on the page template panel.

Reply:
+1 to checking with UX here about that.

If doing that, just make sure it just appears disabled so it's not removed from the accessibility tree.